AbstractSQLCompiler.ts

```diff
@@ -97,28 +97,16 @@ export interface EngineInstance {
 	getModifiedFields: (abstractSqlQuery: AbstractSqlQuery) => undefined | ModifiedFields | Array<undefined | ModifiedFields>,
 }
 
-const validateTypes = _.mapValues(sbvrTypes, ({ validate }) => {
-	if (validate != null) {
-		return Promise.promisify(validate)
-	}
-})
+const validateTypes = _.mapValues(sbvrTypes, ({ validate }) => validate)
```

Comment:
Can;t this be `_.mapValues(sbvrTypes, 'validate')`?

Reply:
Sort of, but I'm pretty sure it messes with types (it also has worse performance but obviously that only matters in some very niche cases)